src/js/core/widget/core/SectionChanger.js

```diff
@@ -212,17 +212,6 @@
 					element.classList.add(classes.uiSectionChanger);
 
 					self.scroller.style.position = "absolute";
-					offsetHeight = element.offsetHeight;
```

Comment:
unused variable `offsetHeight` in line 199.

Reply:
Right. Thanks I will fix this.

Reply:
Fixed.